dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/services/ApiKeyManager.kt

```diff
@@ -72,6 +72,7 @@ class ApiKeyManager(
             null -> null
             else -> {
                 val daysValid = daysValid.toLong()
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/96b24ea7-4d87-415b-946a-b5f1d5d901c4)
Why do we have this fragmented structure for the new framework?

Reply:
The frameworks/gdv directory is wiped completely on every framework-toolbox execution. The model/gdv/ folder contains the code for the human-written custom components for this framework. But you are right. I'll add a subtask to make that more pleasant.

Reply:
Fixed

Reply:
Cool. So now everything of the gdv framework is contained in framework.gdv, correct?

Reply:
Yes. Let me know if you find anything outside of that directory :+1:

Reply:
At least the backend classes, yes.

Reply:
ok, then i will resolve here